### PR TITLE
Helperts tests

### PR DESCRIPTION
**Pull request by ARtoriouSs**

### CODE REVIEW

File: spec/domain/helpers/dates_spec.exs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  let :datetime_without_microseconds, do: ~U[2000-07-21 12:15:03Z]
+
+  let :naive_datetime, do: ~N[2000-07-21 12:15:00]
+
+  describe "date_from_string/1" do
+    it "parses Date from string with '%-d.%-m.%Y' format" do
+      expect Dates.date_from_string(date_string()) |> to(eq date())
+    end
+
+    context "when one-digit date elements passed" do
```
Comment (mirterious) [inline spec/domain/helpers/dates_spec.exs:20]: I think, that we don't need to test it.

File: spec/domain/helpers/permissions_spec.exs [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    member = insert(:squad_member, role: :commander)
+    insert(:user, squad_member: member)
+  end
+
+  let :squad do
+    %{squad_member: %{squad: squad}} = user() |> Repo.preload(squad_member: :squad)
+    squad
+  end
+
+  let :another_squad do
```
Comment (mirterious) [inline spec/domain/helpers/permissions_spec.exs:19]: Probably, this name is not so clear as it necessary
Can we change it to `owned_squad` and `not_owned_squad` or something like that.
Please, share your opinion

**Review by mirterious — CHANGES_REQUESTED** [review]

File: spec/domain/helpers/permissions_spec.exs (hunk 2 above)
Comment (ARtoriouSs) [reply spec/domain/helpers/permissions_spec.exs:19]: maybe not `owned`, e.g. journalist not own the squad

File: spec/domain/helpers/dates_spec.exs (hunk 1 above)
Comment (ARtoriouSs) [reply spec/domain/helpers/dates_spec.exs:20]: As far as it was the bug case, I think it should be tested

**Review by mirterious — APPROVED** [review]

**Pull request merged**